spec/support/api/schemas/song.json

```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "http://spotify-apprentice-app.com",
+  "type": "object",
+  "properties": {
+    "song": {
+      "id": "http://spotify-apprentice-app.com/song",
+      "type": "object",
+      "properties": {
+        "id": {
+          "id": "http://spotify-apprentice-app.com/song/id",
+          "type": "string"
```

Comment:
FYI on JSON schema: if any of these fields *weren't* required, you'd want to update the `type` on that property to be something like:

```
"type": ["string", "null"]
```

otherwise if that field were missing, the test would break. (Also you'd remove that field from the list of required attributes at the bottom.)  json-schema.net just assumes everything is required.